components/blocks/VerticalCards.template.ts

```diff
@@ -0,0 +1,33 @@
+import type { TinaTemplate } from 'tinacms';
+
+export const verticalCardsTemplate: TinaTemplate = {
+  label: 'Vertical Cards',
+  name: 'verticalCards',
+  ui: {
+    previewSrc: '',
+  },
+  fields: [
+    {name: 'title', label: 'Title', type: 'string'},
+    {
+      name: 'cardItems',
+      label: 'Card Items',
+      type: 'object',
+      list: true,
+      ui: {
+        itemProps: (item) => ({
+          key: item.id,
+          label: item.headline,
+        }),
+      },
+      fields: [
+        { name: 'headline', label: 'Headline', type: 'string' },
+        { name: 'description', label: 'Description', type: 'string' },
```

Comment:
if there is too many characters in these fields it will looks bad on the UI :( 
You can avoid this by adding some validation 
https://tina.io/docs/extending-tina/validation/

Reply:
removed description as per designers wants anywhomst